### ISSUE TITLE
Control a motorized pergola lamina roof

### ISSUE THREAD

**Issue by henrikekblad** [rdmtc/node-red-contrib-sun-position#260]
I've read through the documentation but can't find any examples for motorized pergola roofs.

I'd like to control the roof of my pergola to maximize the sun on the terrace on some days. Here's a small YT clip of my setup:
https://www.youtube.com/watch?v=FV0N9mmhCis

Cheers
Henrik

**Comment by Hypnos3** [comment]
That should be just as possible as roller shutters or blinds. Because I don't have something like that, I don't know how to address them with Node-Red.

For the blind controller node, an output in the form of a number can be configured over a wide area. The number for maximally opened and minimally open and the opening steps are adjustable.
Depending on the type of roller shutter, it is possible to set 0-100 if 100 = open and 0 closed or vice versa. Others have 0.00 for open and 1.00 for closed.

Basically, however, any values can be set for open / closed. The numbers for min, max and increment just have to "match".

If the application differs significantly, however, you may also need special translation or implementation translation logic for your specific application. (This will be a little easier with the next version of the node.)

If this is not enough and any other payload is required, the timer node can be used.

**Comment by henrikekblad** [comment]
Ok, I will try to tweak the output to fit my horizontal "blinds" in the roof based on the sun position. Just need the controller-box for the somfy motors to arrive.
